app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/configuration/ConfigurationViewModel.kt

```diff
@@ -46,13 +46,14 @@ class ConfigurationViewModel @Inject constructor(
             port: Int
     ) {
         val address = "ws://$address:$port"
-        dataRepository.listChildren().firstOrError()
-                .flatMapCompletable { list ->
-                    dataRepository.putChildData(list.firstOrNull()?.copy(address = address) ?: ChildDataEntity(address))
-                }
+        dataRepository.doesChildDataExists(address)
+            .flatMapCompletable { exists ->
+                if (exists) Completable.complete()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/636d317b685d72b06abf1b59ef55421cd7244cef/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/configuration/ConfigurationViewModel.kt#L51"></span>Multi-line if statement was found that does not have braces. These should be added to improve readability.</td>
    </tr>
  </tbody>
</table>